vmdb/app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -27,18 +27,14 @@ def prov_field_changed
       changed = (@edit[:new] != @edit[:current])
       render :update do |page|                    # Use JS to update the display
         #Going thru all dialogs to see if model has set any of the dialog display to hide/ignore
-        @edit[:wf].get_all_dialogs.keys.each do |d|
-          page << "li_id = '#{d}_li';"
-          if @edit[:wf].get_dialog(d)[:display] == :show
-            page << "miq_jquery_show_hide_tab(li_id, 'show');"
-          else
-            page << "miq_jquery_show_hide_tab(li_id, 'hide');"
-          end
+        @edit[:wf].get_all_dialogs.each do |dialog_name, dialog|
```

Comment:
@brandondunne worth it to do:
```all_dialogs = get_all_dialogs```
first, then use that in the 2 following lines that call .each ?
If not, I'm ok with this.